apps/yapms/src/lib/icons/Search.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+	let className = '';
+	export { className as class };
+	export let style = '';
+</script>
+
+<svg
+	xmlns="http://www.w3.org/2000/svg"
+	fill="none"
+	viewBox="0 0 24 24"
+	stroke-width="1.5"
+	stroke="currentColor"
+	class={className}
+	{style}
+>
+	<path
+		stroke-linecap="round"
+		stroke-linejoin="round"
+		d="M21 21l-5.197-5.197m0 0A7.5 7.5 0 105.196 5.196a7.5 7.5 0 0010.607 10.607z"
+	/>
+</svg>
```

Comment:
This file should be called MagnifyingGlass. Just a bit more descriptive so you know what it's going to look like while reading the code.

Reply:
Changed!